### ISSUE TITLE
[ANDROID]  fails on error 6777001

### ISSUE THREAD

**Issue by koraysels** [j3k0/cordova-plugin-purchase#868]
```
OSX mojave
Cordova 9
Device: Android 8.1
Plugin Version: 8.0.0
```

### Expected behavior
I can test my purchase by checking its state after I first was able to make a purchase.. 

### Observed behavior
After I was only once (!) succesfull in purchasing my IAP, I cannot list the products anymore. On iOS a can get the listing and see if the 'owned' is true or not. On android there is this error:

```
code: 6777001, message: "Init failed - Failed to query inventory: IabResult…(querying owned items). (response: 6777017:Error)"} 
    code: 6777001
    message: "Init failed - Failed to query inventory: IabResult: Error refreshing inventory (querying owned items). (response: 6777017:Error)"

```

### Steps to reproduce
Just create an In app purchase product, publish it to the play console, release you app in the closed track. Add a tester account to make sure you don;t actually pay for your own tests. buy an in app product (in your app you can call `product.finish()`) And close the app , open it again and refresh the store... Normally you should get all the available products but what you actually get is the above error.. 

I hav eseen multiple questions and issues about this issue, but none of then seem to have an answer..

**Comment by j3k0** [comment]
Duplicate of #76 (did you check and tried all suggested solutions before posting?)

**Comment by koraysels** [comment]
I indeed tried all the suggestions from all the other issues on this repo regarding this error..
It does not resolve on a reboot, it does not resolve when clearing app store cache(which is by the way more complex and deeper hidden in menu's in android 8.1) and I also called finish and configured the fovea veriffy server correctly... The first purchase on a device works but then it errors the entire time..

**Comment by Dexus** [comment]
Maybe you like to share your code, so we can take a look. For me it makes no sense that only the initial order will work and the after not.

**Comment by j3k0** [comment]
`Failed to query inventory` is a setup issue between your app and Google Play. Something is wrong in your setup and that's nothing the plugin can solve for you. Just look up this error on Google for plenty of suggestions.

**Comment by koraysels** [comment]
Mauve it had something to do with the purchaser being a test user? So not a real Google play payment..?

**Comment by j3k0** [comment]
Do you have any error logs from `adb logcat`?

**Comment by koraysels** [comment]
Is It possible that this fails because I haven't configured a bank account to the payment system yet ?
eventhough i am using a testuser account I configured in google play console?

**Comment by mohsinraza** [comment]
I tested your demo project and it works well but when I convert your demo project into vue js with framework 7 it always gives error querying owned items.

Here is my code snippet. let me know if you could help @j3k0
` export default {
components: {},
data() {
return {
newses: [],
productState:'',
title:'',
price:'',
canPurchase:'',
goldCoins:'Gold: ',
isOwned:'',
};
},
mounted() {
const self = this;
self.render();
self.restoreInAppPurchase();

},
methods:{
  restoreInAppPurchase:function(){
    const self = this;

    try{
      if (!window.store) {
        console.log('Store not available');
        return;
      }

      const store = window.store;

      // Enable maximum logging level
      store.verbosity = store.DEBUG;

      store.register(
        {
          id:    'monthly',
          alias: 'my_consumable1',
          type:  store.CONSUMABLE
        }
      );

      store.error(function(error) {
        console.log('ERROR ' + error.code + ': ' + error.message);
      });


      store.when('my_consumable1').updated(self.render);
      //refresh
      store.refresh();
    }catch(err){
      console.log(err);
    }

  },
  render:function(){
    console.log('renderer callded');
    const self = this;
    // Get the product from the pool.
    var product = store.get("my_consumable1");

    if (!product) {
        // $el.html("");
        self.productState = "";
        self.title = "";
        self.price = "";
    }
    else if (product.state === store.REGISTERED) {
        // $el.html("<div class=\"loading\" />");
        self.productState = "loading";
        self.title = "loading";
        self.price = "loading";
    }
    else if (product.state === store.INVALID) {
        // $el.html("");
        self.productState = "Invalid";
        self.title = "";
        self.price = "";
    }
    else {
        // Good! Product loaded and valid.
        // $el.html(
        //       "<div class=\"title\">"       + product.title       + "</div>"
        //     + "<div class=\"description\">" + product.description + "</div>"
        //     + "<div class=\"price\">"       + product.price       + "</div>"
        // );

        self.productState = product.description;
        self.title = product.title;
        self.price = product.price;


        // Is this product owned? Give him a special class.
        if (product.owned){
            self.isOwned = "Owned";
            self.goldCoins = "Gold: 10";
        }else{
          self.isOwned = "Not Owned";
          self.goldCoins = "Gold: 0";
        }

        // Is an order for this product in progress? Can't be ordered right now?
        if (product.canPurchase)
            self.canPurchase = true;
        else
            self.canPurchase = false;
    }


  },
  hide:function(){
    // method called when the view is hidden
        store.off(render);
  }

}
}; `
